src/infinite-hits/__tests__/infinite-hits.spec.ts

```diff
@@ -35,22 +35,103 @@ describe('InfiniteHits', () => {
   });
 
   it('should disable `showMore` button', () => {
     const showMore = jest.fn();
     const fixture = render({ showMore, isLastPage: true });
 
-    const button = fixture.debugElement.nativeElement.querySelector('button');
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadMore'
+    );
     button.click();
 
-    expect(button.disabled).toBeTruthy();
+    expect(button.disabled).toEqual(true);
     expect(showMore).not.toHaveBeenCalled();
     expect(fixture).toMatchSnapshot();
   });
+
+  it('should display `showPrevious()` button', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template: '<ais-infinite-hits [showPrevious]=true></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+
+    expect(button).not.toBeFalsy();
+  });
+
+  it('should display `showPrevious()` button with custom label', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template:
+        '<ais-infinite-hits [showPrevious]=true showPreviousLabel="Load previous"></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+
+    expect(button.innerHTML).toMatchInlineSnapshot(`" Load previous "`);
+  });
+
+  it('should call `showPrevious()` on button click', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template: '<ais-infinite-hits [showPrevious]=true></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+    button.click();
+
+    expect(button.disabled).toEqual(false);
+    expect(showPrevious).toHaveBeenCalled();
+  });
+
+  it('should disable `showPrevious` button', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template: '<ais-infinite-hits [showPrevious]=true></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious, isFirstPage: true });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+    button.click();
```

Comment:
Although we're on the first page, we need to click on the button for it to be disabled?

Reply:
No, it was a copy-past mistake from the previous test, I remove it.